### ISSUE TITLE
spotify-qt macOS build

### ISSUE THREAD

**Issue by andynameistaken** [kraxarn/spotify-qt#121]
Hi, 
Could you provide this instruction for macOS build:
```shell
#!/usr/bin/env bash

# script requires brew qt5
mkdir build
cd build
cmake -DQt5_DIR=$(brew --prefix qt5)/lib/cmake/Qt5 ..
make
cd ..
ln -s build/spotify-qt spotify-qt
cd ..
mv spotify-qt spotify-qt.app
mv spotify-qt.app /Applications
```

It works for macOS Catalina, there is one caveat: it doesn't have icon because I don't know how macOS deals with them.

**Comment by andynameistaken** [comment]
Also if you wish I can share  built binary with you, so you can put it into releases

**Comment by kraxarn** [comment]
I've chosen not to do automatic builds for macOS as the binaries require signing to be used outside the device it was compiled on. The instructions are too similar to the general builds instructions to explicitly specify it, but including more detailed building instructions for various platforms could be documented in the wiki.

As for icons, macOS applications are usually contained within a folder suffixed with ".app", making macOS treat it as a binary application. Within that folder, the binary itself is contained, together with the icon, signature, etc.

**Comment by kraxarn** [comment]
Fixed in e3f09360e333fe1b7844d3de09585cd0797cab87, although, they are currently unsigned and not well-tested.
